man/org.freedesktop.machine1.xml

```diff
@@ -112,10 +112,12 @@ node /org/freedesktop/machine1 {
                        in  b mkdir);
       CopyFromMachine(in  s name,
                       in  s source,
-                      in  s destination);
+                      in  s destination,
+                      in  b force_copy);
       CopyToMachine(in  s name,
                     in  s source,
-                    in  s destination);
+                    in  s destination,
+                    in  b force_copy);
```

Comment:
dbus methods are API, and hence you cannot just add or remove params, because that would break API.

Hence, instead of changing the existing method, please add CopyToMachineWithFlags() which is like CopyToMachine(), but takes the extra flags param